libs/components/forms/src/lib/modules/form-error/form-error.component.ts

```diff
@@ -49,6 +50,10 @@ export class SkyFormErrorComponent {
   @Input({ required: true })
   public errorText!: string;
 
+  @HostBinding('attr.errorName') public get hostErrorName(): string {
+    return this.errorName;
+  }
+
```

Comment:
Can we add a unit test in the test harness to make sure this bug scenario is covered

Reply:
added